pkgs/desktops/gnome/core/libgnome-keyring/default.nix

```diff
@@ -13,9 +13,23 @@ stdenv.mkDerivation (finalAttrs: {
 
   outputs = [ "out" "dev" ];
 
+  strictDeps = true;
   propagatedBuildInputs = [ glib gobject-introspection dbus libgcrypt ];
   nativeBuildInputs = [ pkg-config intltool ];
```

Comment:
Maybe we should try to use `autoreconfHook`?

Reply:
I do not want to touch this. It should die already.

Reply:
> I do not want to touch this. It should die already.

Oh I think the 1st commit's title is misleading - you are in fact editing the expression of the attribute `libgnome-keyring`, and not `libgnome-keyring3`.

Reply:
It is modifying `gnome.libgnome-keyring` (`libgnome-keyring3`) at the time. The package is only renamed later.

Reply:
> I do not want to touch this. It should die already.

This is the comment that confused me BTW - why should it die?

Reply:
It adds extra maintenance work just to support few apps that could not be bothered to migrate to `libsecret` in the 10+ years libgnome-keyring has been deprecated.

Reply:
> It adds extra maintenance work just to support few apps that could not be bothered to migrate to `libsecret` in the 10+ years libgnome-keyring has been deprecated.

Ahh OK so `libgnome-keyring` is replaced by `libgnome-keyring3` which should really be replaced by `libsecret`.

Still though, if we don't have that many packages still depending on `libgnome-keyring` or `libgnome-keyring3`, I'd make the effort to try to add `autoreconfHook`, but I don't care too much about this old library to insist on that :).